contracts/icns-registrar/src/contract.rs

```diff
@@ -39,6 +39,7 @@ pub fn instantiate(
     // validate name nft address
     let name_nft_addr = deps.api.addr_validate(&msg.name_nft_addr)?;
 
+    // NL: Should this be disabled?
```

Comment:
This should not, disabled due to `k256` crate has `rand`, will re-enable it as sanity check function. Or do you have better option?

Reply:
I think that makes sense. Just a sanity check on the length to avoid typos etc